lib/portlayer/storage/vsphere/export.go

```diff
@@ -156,7 +163,10 @@ func (i *ImageStore) Export(op trace.Operation, id, ancestor string, spec *archi
 		return nil, err
 	}
 
-	return tar, nil
+	return &storage.ProxyReadCloser{
```

Comment:
Side question: are we still using the CleanupReader? If not, we could probably remove the file if it hasn't been already.